### DISCUSSION TITLE
Setup klipperscreen on second raspberry?

### DISCUSSION THREAD

**Discussion by pdSquirrel** [KlipperScreen/KlipperScreen#574] [Q&A]
Cant seem to find good documentation on how to setup klipperscreen on a secondary raspberry, connecting to the first one that is running klipper. Do I still need to install klipper on the second raspberry or is it enough with moonraker and klipperscreen and then point to my real klipper instance on raspberry number one?

Please, can someone enlighten me :)

**Comment by alfrix** [comment]
you don't need klipper or moonraker installed with klipperscreen, the only thing needed is adding a printer section in the conifg with the correct ip and port of where moonraker is installed
